app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDeclarationDetailsController.scala

```diff
@@ -342,7 +342,9 @@ class EnterDeclarationDetailsController @Inject() (
                     )
                   )
                 case Right(_)          =>
-                  Redirect(routes.EnterMovementReferenceNumberController.enterDuplicateMrn()) //FIXME double check this
+                  Redirect(
+                    routes.EnterDuplicateMovementReferenceNumberController.enterDuplicateMrn()
+                  ) //FIXME double check this
```

Comment:
Why there is //FIXME double check this
Shall it be just removed?

Reply:
Removed.